### DISCUSSION TITLE
chapter 4, page 27, exercise set 4.2, Q13

### DISCUSSION THREAD

**Discussion by Zayd-R** [spamegg1/Epp-Discrete-Math-5th-solutions#4] [Q&A] [answered]
6n + 27 should be 6n^2 + 27

I do not know If you care about these types of mistakes so I will take your feedback on it and ignore any similar mistakes if you do not care about them. 

Thank you for your work.

**Comment by spamegg1** [comment, marked as answer]
Thanks! It does not affect the solution but I definitely care, so keep 'em coming. Fixed [here](https://github.com/spamegg1/Epp-Discrete-Math-5th-solutions/commit/f5a88ea20c65527c620bbb615bbebb5be6e87833#diff-9c927f29acb54df45a20f356a9c52e7aa666bffcad0fc6224705242d8cdb4422)
